controllers/contentlibrary/utils/constants.go

```diff
@@ -11,8 +11,10 @@ const (
 	ContentLibraryKind            = "ContentLibrary"
 	ContentLibraryItemKind        = "ContentLibraryItem"
 
-	ContentLibraryItemVmopFinalizer        = "contentlibraryitem.vmoperator.vmware.com"
-	ClusterContentLibraryItemVmopFinalizer = "clustercontentlibraryitem.vmoperator.vmware.com"
+	CLItemFinalizer            = "vmoperator.vmware.com/contentlibraryitem"
+	DeprecatedCLItemFinalizer  = "contentlibraryitem.vmoperator.vmware.com"
+	CCLItemFinalizer           = "clustercontentlibraryitem.vmoperator.vmware.com"
+	DeprecatedCCLItemFinalizer = "vmoperator.vmware.com/clustercontentlibraryitem"
```

Comment:
@aruneshpa I think there is a typo for `CCLItemFinalizer`. The new name should contain a slash, and the old (deprecated name) is only a domain. The other Finalizers look good.